cli/tsc/mod.rs

```diff
@@ -586,7 +590,7 @@ fn op_resolve(
     let resolved_dep = graph
       .get(&referrer)
       .and_then(|m| m.esm())
-      .and_then(|m| m.dependencies.get(&specifier))
```

Comment:
Does this line applies fast check for all type checking in the CLI?

Reply:
Yeah, only if it appears in the graph.

Reply:
Wow, positively surprising that no other test failed!

Reply:
This internally does `if let Some(module) = &self.fast_check { &module.dependencies } else { &self.dependencies }`.